tests/auth_tests/urls.py

```diff
@@ -78,7 +78,7 @@ def login_and_permission_required_exception(request):
     pass
 
 
-uid_token = r'(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})'
+uid_token = r'(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,32})'
```

Comment:
Should we just make this `+` instead of `{1,32}` -- we'd just have to change it gain if the algo changes again,